### ISSUE TITLE
Highlighting breaks with question marks in Vue template attributes

### ISSUE THREAD

**Issue by mattjneuber** [octref/vetur#3]
```html
<template>
	<element :statechanged="(a == b) ? x : y"></element>
	<span :class="{'thing': true}"></span>
</template>
```

produces [this output](http://i.imgur.com/FSG40ti.png)

Removing the ? from the expression immediately fixes the highlighting after it.

**Comment by octref** [comment]
Thanks for reporting. Will take a look after work.

**Comment by octref** [comment]
Upstream bug tracked at https://github.com/Microsoft/vscode/issues/15289
The last double quote get parsed as JavaScript double quote, so VSCode thought everything after it is a JavaScript string.

**Comment by octref** [comment]
I just published a new version 0.1.1 including the fix.

However, this problem no longer exists on latest Insider (still repro on stable), so I guess they updated the JS grammar and fixed the problem. I'll look into it and see if I can also adopt that grammar.
